src/main/java/org/broadinstitute/hellbender/tools/walkers/qc/CheckPileup.java

```diff
@@ -0,0 +1,226 @@
+package org.broadinstitute.hellbender.tools.walkers.qc;
+
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.QCProgramGroup;
+import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.IntervalUtils;
+import org.broadinstitute.hellbender.utils.codecs.sampileup.SAMPileupFeature;
+import org.broadinstitute.hellbender.utils.pileup.ReadPileup;
+
+import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Compare GATK's internal pileup to a reference Samtools pileup
+ *
+ * <p>At every locus in the input set, compares the pileup data (reference base, aligned base from
+ * each overlapping read, and quality score) generated internally by GATK to a reference pileup data generated
+ * by Samtools. Note that the pileup program has been replaced in Samtools by mpileup, which produces a slightly
```

Comment:
Does your tool read mpileup format?  Samtools no longer offers the pileup command.  It looks like mpileup produces something very similar to the 6 column format here, but I'm not sure what the differences if any are.